school.rb

```diff
@@ -0,0 +1,51 @@
+class School
+attr_reader :age, :salary, :phase, :target_raise
+attr_accessor :name
```

Comment:
Indent lines 2 and 3.  

Also, you have student inheriting from school, but originally, students didn't have `salary` or `target_raise` (and it doesn't make sense for them to have those attributes).  Push those down into the `Instructor` class.